### PR TITLE
templates: welcome news

### PR DESCRIPTION
**Pull request by atrisovic**
Signed-off-by: Ana Trisovic <ana.trisovic@gmail.com>

### CODE REVIEW

File: cernopendata/modules/fixtures/data/articles/news/welcome.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+With the growing demand of open science data and increased interest in the CERN Open Data portal, we were working hard on modernizing our web presence. Here, we are telling you more about the changes we made and how you can use the updated portal.
+
+<img src="../../../../static/assets/header-image.svg" width="70%" align="middle">
+
+CERN Open Data provides content for both education and research. We aim to support high school students and teachers as well as university students and professors. For that purpose, the new design of CERN Open Data is article-driven. Exciting guides introduce to physics analyses and make use of datasets. Event displays allow to visualise and imagine what happens during collisions of the smallest elements.
```
Comment (sefeg) [inline cernopendata/modules/fixtures/data/articles/news/welcome.md]: @atrisovic could you please add the relative URLs that I added to certain words?

Comment (sefeg) [reply cernopendata/modules/fixtures/data/articles/news/welcome.md]: I like the "FIXME PRELIMINARY TEXT" attribution @tiborsimko referred to. It should be a visible starting point that presents a base for collaborative improvement.

Comment (atrisovic) [reply cernopendata/modules/fixtures/data/articles/news/welcome.md]: Ok, but I don't see the relative urls in the google file (there is only one link for the email)

**Comment by atrisovic** [comment]
OK! @tiborsimko

**Pull request closed without merging**